app/views/state_file/questions/nj_sales_use_tax/edit.html.erb

```diff
@@ -0,0 +1,51 @@
+<% title = t('.title') %>
+<% content_for :page_title, title %>
+
+<% content_for :card do %>
+  <h1 class="h2"><%= title %></h1>
+  <p><%= t(".subtitle_html") %></p>
+  <%= form_with model: @form, url: { action: :update }, local: true, method: "put", builder: VitaMinFormBuilder do |f| %>
+    <div class="question-with-follow-up spacing-below-25">
+      <div class="question-with-follow-up__question">
+        <div class="white-group">
+          <%=
+            f.cfa_radio_set(
+              :untaxed_out_of_state_purchases,
+              label_text: t('.use_tax_radio_label'),
+              collection: [
+                { value: :yes, label: t("general.affirmative"), input_html: { "data-follow-up": "#sut-method-field" } },
+                { value: :no, label: t("general.negative") },
```

Comment:
[DUST] How does the `sut-method-field` get shown or hidden? I see a way to hide the `#sut-field` onclick
```
input_html: { "onclick": "document.querySelector('#sut-field').style.display = 'none'"} },
```

but not for this first question. How is that visibility controlled?

Reply:
It's controlled by the honeycrisp classes and attributes. `question-with-follow-up` and the `"data-follow-up": "#sut-method-field"` on the initial Yes radio button

Reply:
If you want to see how it works you can check out the `followUpQuestion` function in the `honeycrisp.js` file